query/query_test.go

```diff
@@ -0,0 +1,28 @@
+package query
+
+import (
+	. "github.com/smartystreets/goconvey/convey"
+	"testing"
+)
+
+func TestSetupSearch(t *testing.T) {
+	Convey("Should return templates and error should be nil", t, func() {
+
+		searchTemplates, err := SetupSearch("../")
+
+		So(err, ShouldBeNil)
+		So(searchTemplates, ShouldNotBeNil)
+
+	})
+}
+
+func TestNewQueryBuilder(t *testing.T) {
+	Convey("Should return a Builder object with templates", t, func() {
+		pathToTemplates := "../"
```

Comment:
Probably worth keeping each test consistent with how you set the value pathToTemplates.

A global variable that is not named `pathToTemplates` in test so it doesn't conflict with the actual code:

```
var testPathToTemplates = "../"

...

func TestSetupSearch(t *testing.T) {
	Convey("Should return templates and error should be nil", t, func() {

		searchTemplates, err := SetupSearch(testPathToTemplates)

...

func TestNewQueryBuilder(t *testing.T) {
	Convey("Should return a Builder object with templates", t, func() {
		builderObject, err := NewQueryBuilder(testPathToTemplates)
...
```